docs/advanced.md

```diff
@@ -0,0 +1,20 @@
+# Advanced configuration
+
+Congrats on getting `flow-typed` set up! There are a small number of configuration options available through the CLI, but using all those flags can be tiring. This tiny guide will walk through setting up custom scripts in `package.json`.
+
+## Custom Scripts
+
+Before editing your `package.json`, take a look at the available [[CLI Commands & Flags|CLI-Commands-and-Flags]]. Create an entry for each command you'd like to customize.
```

Comment:
Did you mean for `[[CLI Commands & Flags|CLI-Commands-and-Flags]]` to link somewhere?